integration-tests/validation.py

```diff
@@ -79,9 +80,61 @@ def validate_status_messages(model):
                 assert snap_version.startswith(track + '.')
 
 
+async def validate_rbac(model):
+    ''' Validate RBAC is actually on '''
+    app = model.applications['kubernetes-master']
+    await app.set_config({'authorization-mode': 'RBAC,Node'})
+    await wait_for_process(model, 'RBAC')
+    cmd = "/snap/bin/kubectl --kubeconfig /root/cdk/kubeconfig get clusterroles"
+    worker = model.applications['kubernetes-worker'].units[0]
+    output = await worker.run(cmd)
+    assert output.status == 'completed'
+    assert "forbidden" in output.data['results']['Stderr']
+    await app.set_config({'authorization-mode': 'AlwaysAllow'})
+    await wait_for_process(model, 'AlwaysAllow')
+    output = await worker.run(cmd)
+    assert output.status == 'completed'
+    assert "forbidden" not in output.data['results']['Stderr']
+
+
+async def validate_rbac_flag(model):
+    ''' Switch between auth modes and check the apiserver follows '''
+    master = model.applications['kubernetes-master']
+    await master.set_config({'authorization-mode': 'RBAC'})
```

Comment:
Does this one need Node too?

Reply:
I am just checking that the config is reflected on the kubeapi process. For this we can use any valid value. I have here only RBAC since RBAC+Node is indirectly tested by the previous test.